flexget/utils/imdb.py

```diff
@@ -321,7 +322,13 @@ def parse(self, imdb_id):
         else:
             log.debug('No score found for %s' % self.imdb_id)
 
-        # get director(s)
+        meta_score_elem = title_overview.find(attrs={'class': 'metacriticScore'})
+        if meta_score_elem:
+            self.meta_score = str_to_int(meta_score_elem.text)
+        else:
+            log.debug('No Metacritic score found for %s' % self.imdb_id)
```

Comment:
This line should be changed to `log.debug('No Metacritic score found for %s', self.imdb_id)`

Reply:
Does that mean that all of the other element debug statements need to be corrected as well? (e.g. score_elem, votes_elem, original_name_elem, photo_elem, mpaa_rating_elem, etc.) They are all formatted as I have it for meta_score_elem.  If this is the case I can fix those as well.

Reply:
Yes, it's best to let the logger handle the string interpolation.